pkgs/unified_analytics/lib/src/event.dart

```diff
@@ -4,8 +4,12 @@
 
 import 'dart:convert';
 
+import 'package:collection/collection.dart';
```

Comment:
let's try to avoid adding any direct dependencies to this package, as it complicates rolling to flutter/flutter (dev_dependencies are ok).